### PR TITLE
DMP-2221: Checkin correction to SOAPUI. Also added postman equivalent file (exl…

### PR DESCRIPTION
**Pull request by mestebanez**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DMP-2221

### Change description ###

Small corrections to the SOAP UI project. Additionally add an equivalent postman project. SOAP UI is unusable on mac silicon:-

https://tools.hmcts.net/jira/browse/DMP-2221

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,31 @@ following properties within SoapUI
...
+* [Postman](https://www.postman.com/)  can be used for "Try it out" functionality
+  using the ServiceContext Header with a valid system user (CPP / XHIBIT / MID_TIER). Firstly you will need to set the
+  following properties within Postman NOTE: This excludes the add audio test. SoapUi must be used in the case of addaudio
+
+  *  userToUse - The user to authenticate with
+  *  passwordToUse - The password to authenticate with
+  *  tokenToUse - The token (jwt by default) to authenticate with
+  *  gatewayurl - The gateway url to use
+
+## DARTSService SoapUI
```
Comment (richardbull) [inline README.md:47]: Typo - these 2 subheaders probably need to end with Postman instead of SoapUI?

File: README-DARTSService-soapui-project.xml
Comment (richardbull) [inline README-DARTSService-soapui-project.xml:1]: Imported this into SoapUI and tried Validate on `audioWithToken` request.
The document in the body doesn't appear to be valid.

Comment (richardbull) [inline README-DARTSService-soapui-project.xml:1]: Prefer that we use the Format XML option on the request to pretty-print the example - just to make it a bit easier to read.

I think you can do this with the document XML stuff before you have to encode it as a string xml or use CDATA tags around it.

[the remark this answers is not in the document]
Comment (mestebanez) [reply README-DARTSService-soapui-project.xml:1]: They get taken into account if none are set at the project level but I will add this approach anyhow

[the remark this answers is not in the document]
Comment (mestebanez) [reply README-DARTSService-soapui-project.xml:1]: to be honest we should move away from SOAPUI, its pedegree doesnt seem to be great these days

**Review by richardbull — APPROVED** [review]

**Pull request merged**